config/allowlistlist.go

```diff
@@ -36,6 +36,64 @@ type Allowlist struct {
 	Subnets Subnets `json:"subnets"`
 }
 
+func (a *Allowlist) UpdateUDPPorts(ports []int64, remove bool) {
```

Comment:
Why use have here `int64` when `uint16` is enough for values in range `1...65535`?

Reply:
`int64` is used all throughout the app to represents ports, so I wanted to maintain compatibility. I'll add a task to tech debt.